editor/src/components/canvas/__snapshots__/ui-jsx-canvas-bugs.spec.tsx.snap

```diff
@@ -11,7 +11,7 @@ exports[`UiJsxCanvas #747 - DOM object constructor cannot be called as a functio
   \\"
 >
   <div
-    data-utopia-scene-id=\\"fc3/12b\\"
+    data-utopia-scene-id=\\"32d/0f1\\"
```

Comment:
interesting – these changed without a change to ui-jsx-canvas-bugs.spec. is this because our hash generation changed?

Reply:
Hmmm, the hash generation itself hasn't changed, I would surmise this is due to comments being included in the representation of the JSX attributes.